Src/BootCamp.Chapter/MsFactory.cs

```diff
@@ -5,11 +5,61 @@
 
 namespace BootCamp.Chapter
 {
-    public class MsFactory
+    public class MsFactory : ComputerFactory
     {
-        public DesktopComputer Assemble()
+        public override DesktopComputer Assemble()
         {
-            return new DesktopComputer();
+            var computer = new DesktopComputer();
+            TakeBody(computer);
+            PlaceMotherBoard(computer);
+            PlaceHarddisk(computer);
+            PlaceCpu(computer);
+            PlaceGpu(computer);
+            PlaceRam(computer);
+            Console.WriteLine("the ms computer is ready to be shipped");
+            return computer;
+        }
+
+        public override void PlaceRam(DesktopComputer computer)
+        {
+            Console.WriteLine("Im going to place a standard ram");
+            var ram = new Ram("a  standard cpu");
+            computer.SetRam(ram);
+        }
+
+        public override void PlaceGpu(DesktopComputer computer)
+        {
+            Console.WriteLine("Im going to place a standard gpu");
+            var gpu = new Gpu("a  standard cpu");
+            computer.SetGpu(gpu);
+        }
+
+        public override void PlaceCpu(DesktopComputer computer)
+        {
+            Console.WriteLine("Im going to place a MS cpu");
+            var cpu = new Cpu("a  MS cpu");
+            computer.SetCpu(cpu);
+        }
+
+        private void PlaceHarddisk(DesktopComputer computer)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'PlaceHarddisk' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5124353)

Reply:
done